<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="NewAction"/>
    <addaction name="OpenAction"/>
    <addaction name="SaveAction"/>
   </widget>
   <widget class="QMenu" name="EditMenu">
    <property name="title">
     <string>编译(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="BuildMenu">
    <property name="title">
     <string>构建(&amp;B)</string>
    </property>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="EditMenu"/>
   <addaction name="BuildMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="NewAction">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="OpenAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="text">
    <string>另存为(&amp;S)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
